tasks/skylab/StarAlign.wdl

```diff
@@ -100,21 +104,60 @@ task StarAlignFastqPairedEnd {
   command {
     set -e
 
+    UMILen=10
+    CBLen=16
+    if [ "${chemistry}" == "tenX_v2" ]
+    then
+        ## V2
+        UMILen=10
+        CBLen=16
+    elif [ "${chemistry}" == "tenX_v3" ]
+    then
+        ## V3
+        UMILen=12
+        CBLen=16
+    else
+        echo Error: unknown chemistry value: "$chemistry"
```

Comment:
Like in line 135, should we update this error message to say "should be 10X_v2 or 10X_v3" or whatever the options are?

Reply:
updated